cli/init/templates/tt.yaml.default

```diff
@@ -0,0 +1,62 @@
+tt:
+  modules:
+    # Directory where the external modules are stored.
+    directory: "{{ .CliConfig.Modules.Directory }}"
+
+  app:
+    # Directory that stores various instance runtime
+    # artifacts like console socket, PID file, etc.
+    run_dir: "{{ .CliConfig.App.RunDir }}"
+
+    # Directory that stores log files.
+    log_dir: {{ .CliConfig.App.LogDir }}
+
+    # The maximum size in MB of the log file before it gets rotated.
+    log_maxsize: {{ .CliConfig.App.LogMaxSize }}
+
+    # The maximum number of days to retain old log files.
+    log_maxage: {{ .CliConfig.App.LogMaxAge }}
+
+    # The maximum number of old log files to retain.
+    log_maxbackups: {{ .CliConfig.App.LogMaxBackups }}
+
+    # Restart instance on failure.
+    restart_on_failure: {{ .CliConfig.App.Restartable }}
+
+    # Directory where write-ahead log (.xlog) files are stored.
+    wal_dir: "{{ .CliConfig.App.WalDir }}"
+
+    # Directory where memtx stores snapshot (.snap) files.
+    memtx_dir: "{{ .CliConfig.App.MemtxDir }}"
+
+    # Directory where vinyl files or subdirectories will be stored.
+    vinyl_dir: "{{ .CliConfig.App.VinylDir }}"
+
+    # Directory that stores binary files.
+    bin_dir: "{{ .CliConfig.App.BinDir }}"
+
+    # Directory that stores Tarantool header files.
+    inc_dir: "{{ .CliConfig.App.IncludeDir }}"
+
+    # Path to directory that stores all applications.
+    # The directory can also contain symbolic links to applications.
+    instances_enabled: "{{ .CliConfig.App.InstancesEnabled }}"
+
+    # Tarantoolctl artifacts layout compatibility: if set to true tt will not create application
+    # sub-directories for control socket, pid files, log files, etc.. Data files (wal, vinyl,
+    # snap) and multi-instance applications are not affected by this option.
+    tarantoolctl_layout: {{ .CliConfig.App.TarantoolctlLayout }}
+
+  # Path to file with credentials for downloading Tarantool Enterprise Edition.
+  # credential_path: /path/to/file
+  ee:
+    credential_path: "{{ .CliConfig.EE.CredPath }}"
+
+  templates:
+    # The path to templates search directory.{{ $templates := .CliConfig.Templates }}{{ range $_, $template := $templates }}
+    - path: "{{ $template.Path }}"{{end}}
+
+  repo:
+    # Directory that stores installation files.
```

Comment:
This comment is about distfiles, not rocks.

Reply:
Fixed.